DiceKeys/Views/Scanning/ScanDiceKey.swift

```diff
@@ -21,11 +21,14 @@ struct ScanDiceKey: View {
     init(stickers: Bool = false, onDiceKeyRead: ((_ diceKey: DiceKey) -> Void)? = nil) {
         self.stickers = stickers
         self.onDiceKeyRead = onDiceKeyRead
+        getAvailableCameras()
     }
 
     @State var frameCount: Int = 0
     @State var facesRead: [FaceRead]?
     @State var processedImageFrameSize: CGSize?
+    @State var selectedCameraIndex: Int = 0
+    private var availableCameras = [AVCaptureDevice]()
```

Comment:
Non-state mutated properties are best avoided.

Why not make it a computed property?

Reply:
I prefer avoiding computing properties whenever I can. It is not an issue right now but as the app grows too many computed properties make their way and it becomes very slow.

Anyway, in this case I didn't think we had to update camera list in real time.